src/components/CarrouselProfile/index.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+//Imports Material-ui
+import { Box, Typography } from "@material-ui/core";
+//Imports Components
+import CardCarrousel from "components/CardCarrousel";
+//Imports Styles
+import "./styles.scss";
+
+const CarrouselProfile = ({ titleSection, data }) => {
```

Comment:
Cambiaría para hacerlo más significativo `data` por `items`, que es lo que un `Carrousel` contiene, además, es en plural, lo que da una mejor idea del tipo de información que espera

```suggestion
const CarrouselProfile = ({ titleSection, items }) => {
```

Reply:
Done!